.github/PULL_REQUEST_TEMPLATES/add_update_component.md

```diff
@@ -0,0 +1,22 @@
+## 🎯 Changes
+
+<!-- Describe the changes made in this PR. Is it a new component, tool, or template? Or is it an update to an existing one? -->
```

Comment:
```suggestion
<!-- Describe the changes made in this PR. Is it a new package or template? Or is it an update to an existing one? -->
```
The distinction between component and tool have been removed.
There are only two kind of content;
- package (before it was component and tools)
- template
- recipe
- cheat sheet
- event
- resource (video, books)